Resources/doc/multiple_file_upload.md

```diff
@@ -0,0 +1,59 @@
+# Multiple file upload
+
+In order to use multiple file upload feature you must add a listener to collection holding file forms. When instantiating listener pass name of field that holds file instance.
+
+```php
+$formBuilder->get('files')->addEventSubscriber(new MultiUploadCollectionListener('file'));
```

Comment:
It's no longer necessary to add subscriber. Now it's enough to set ``multi_upload_field`` option on collection form.